pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -282,7 +282,7 @@ in with passthru; stdenv.mkDerivation {
 
   CPPFLAGS = concatStringsSep " " (map (p: "-I${getDev p}/include") buildInputs);
   LDFLAGS = concatStringsSep " " (map (p: "-L${getLib p}/lib") buildInputs);
-  LIBS = "${optionalString (!stdenv.isDarwin) "-lcrypt"} ${optionalString (ncurses != null) "-lncurses"}";
+  LIBS = "${optionalString (!stdenv.isDarwin) "-lcrypt"}";
```

Comment:
Crazy idea: we may even get rid of the entire line.

```suggestion
```

Original:
```console
$ nix-build -A python39
$ ./result/bin/python3.9-config --libs
 -lpthread -ldl -lcrypt -lncurses -lutil -lm -lm
```

with ncurses removed:
```console
$ nix-build -A python39
$ ./result/bin/python3.9-config --libs
 -lpthread -ldl -lcrypt -lutil -lm -lm
```

with ncurses and crypt removed:
```console
$ nix-build -A python39
$ ./result/bin/python3.9-config --libs
 -lcrypt -lpthread -ldl  -lutil -lm -lm
```

This is the same as Debian 11 produces:

```console
$ sudo apt install python3-dev
$ python3-config --libs
 -lcrypt -lpthread -ldl  -lutil -lm -lm
```

Reply:
Let's keep that for a separate PR.